Apromore-Custom-Plugins/Log-Animation-Portal-Plugin/src/main/java/org/apromore/plugin/portal/loganimation/LogAnimationCleaner.java

```diff
@@ -40,7 +40,7 @@
 
     @Override
     public void cleanup(Desktop desktop) throws Exception {
-        LOGGER.info("LogAnimation cleanup starts for desktopID = " + desktop.getId());
+        LOGGER.debug("LogAnimation cleanup starts for desktopID = {}", desktop.getId());
         
         // Clean up this plugin
         // Log animation logic has no tricky resources to be cleaned up, auto cleaned by the GC
```

Comment:
This should be INFO since it marks the end of the plugin execution.

Reply:
This is precisely the spammy, non-user-initiated internal detail that we're trying to remove from INFO level.  It needs to be logged, and it gets logged at DEBUG level.  The system administrator does not want to see session management details like this.  They just care that the user logged out, and that gets logged at INFO level.

Reply:
Reinstated at INFO level.